relay-server/src/utils/sizes.rs

```diff
@@ -63,6 +63,9 @@ pub fn check_envelope_size_limits(config: &Config, envelope: &Envelope) -> bool
             ItemType::UserReport => (),
             ItemType::Metrics => (),
             ItemType::MetricBuckets => (),
+            // The Combined Replay Envelope isn't generated on the client so its size does not need
+            // to be checked.
+            ItemType::CombinedReplayEventAndRecording => (),
```

Comment:
Even if envelopes are generated in Relays, size requirements may vary and we don't have control over external relays, which could be running an older version and sending bigger envelopes.

Reply:
gotcha, will add a size here.